openshift/app.dc.yaml

```diff
@@ -252,6 +252,8 @@ objects:
                     name: "${APP_NAME}-service-config"
                 - configMapRef:
                     name: "${APP_NAME}-files-config"
+                - configMapRef:
+                    name: "${APP_NAME}-oidc-config"
```

Comment:
With this new ConfigMap can the `SERVER_KC_CLIENTID` and `SERVER_KC_CLIENTSECRET` environment variables in this DeploymentConfig be removed?

Reply:
yes, all the XXX_KC_YYY env vars can go away. And there will be no more KC secret or public key